t/test-data/Models/hr-inclNOxExtendedWithFilters/Livestock/DairyCow/Excretion.nhd

```diff
@@ -87,26 +87,29 @@ Flisch R, Sinaj S, Charles R, Richner W 2009. Grundlagen für die die Düngung i
        <p>Nombre effectif d'animaux dans la stabulation.</p>
 
 +dimensioning_barn
-  type = float
+  type = text
   distribute = true
+  default_calc = Standard
+  default_gui  = Standard
+  default_formula = In(animals);
```

Comment:
I think `default_calc` and `default_formula` should be mutually exclusive (and in the future we should start to detect that); thus the `default_calc` should removed here (`default_gui` should stay along with `default_formula`, however).

Reply:
Has been removed in c1a8c29.